Revamped Overview Articles (#1131)
telerik/blazor-docs: components/gauges/arc/overview.md

The <a href = "https://www.telerik.com/blazor-ui/arc-gauge" target="_blank">Telerik Arc Gauge for Blazor</a> represents [numerical values]({%slug arc-gauge-pointers%}) on an arc [scale]({%slug arc-gauge-scale%}).

> ## Creating Blazor Arc Gauge

Reviewer: Consider a bit more real-world example, e.g. bind the `Value` to some property. 

```razor
<TelerikArcGauge>
    <ArcGaugePointers>
        <ArcGaugePointer Value="@GaugeValue" />
    </ArcGaugePointers>
</TelerikArcGauge>

@code {
    private double GaugeValue { get; set; } = 40;
}
```

1. Add the `<TelerikArcGauge>` tag.

1. Add an instance of the `<ArcGaugePointer>` to the `<ArcGaugePointers>` collection.

1. Provide a `Value` for the `<ArcGaugePointer>`.

1. (optional) Add a [Center Label Template]({%slug arc-gauge-labels%}#center-template)

> >caption Basic Telerik Arc Gauge for Blazor.

Reviewer: We are removing all unnecessary screenshots, except those, which:
- demonstrate some user interaction via animation
- include annotations and embedded explanations

## Pointers

The pointers indicate the values on the scale of the component. See the [Pointers]({%slug arc-gauge-pointers%}) article for more information on how to customize the pointers of the component.

## Labels

The labels are rendered on the scale of the component to give information to the users. See the [Labels]({%slug arc-gauge-labels%}) article for more information on how to customize the labels on the scale of the component.

## Arc Gauge Parameters

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
| Parameter | Type and Default Value | Description |
| --- | --- | --- |
| `Class` | `string` | Renders a custom CSS class to the `<div class="k-arcgauge">` element. |
| `Width` | `string` | Controls the width of the Arc Gauge. You can read more information in the [Dimensions]({%slug common-features/dimensions%}) article.|
| `Height` | `string` | Controls the height of the Arc Gauge. You can read more information in the [Dimensions]({%slug common-features/dimensions%}) article. |

> | `Transitions` | `bool?` | Controls if the animation transitions must be rendered. |

Reviewer: ```suggestion
| `Transitions` | `bool?` | Controls if the Arc Gauge uses animations for its value changes. |
```

</TelerikArcGauge>
````

## Scale

> The scale of the arc gauge renders the values of the [pointers]({%slug arc-gauge-pointers%}) and [labels]({%slug arc-gauge-labels%}). See the [Scale]({%slug arc-gauge-scale%}) article for more information on how to customize the scale of the component.

Reviewer: Prefer capital letters when talking about the component as a product. Also consider mentioning some preliminary information, for example - the scale allows the developer to configure min/max values, angles and ticks. This will increase the value of the Overview article and you can even use more relevant keywords for the article link. See the article links at https://docs.telerik.com/blazor-ui/components/scheduler/overview

```suggestion
The scale of the Arc Gauge renders the values of the [pointers]({%slug arc-gauge-pointers%}) and [labels]({%slug arc-gauge-labels%}). See the [Scale]({%slug arc-gauge-scale%}) article for more information on how to customize the scale of the component.
```

## Pointers

The pointers indicate the values on the scale of the component. See the [Pointers]({%slug arc-gauge-pointers%}) article for more information on how to customize the pointers of the component.

## Labels

The labels are rendered on the scale of the component to give information to the users. See the [Labels]({%slug arc-gauge-labels%}) article for more information on how to customize the labels on the scale of the component.

## Arc Gauge Size

<style>
    article style + table {
        table-layout: auto;
        word-break: normal;
    }
</style>

> | Parameter | Type and Default Value | Description |

Reviewer: Add the `Transitions` parameter. It is important, because avoids the repetitive animation on value change, which always starts from the `Min` value.

| --- | --- | --- |
| `Class` | `string` | Renders a custom CSS class to the `<div class="k-arcgauge">` element. |
| `Width` | `string` | Controls the width of the Arc Gauge. You can read more information in the [Dimensions]({%slug common-features/dimensions%}) article.|
| `Height` | `string` | Controls the height of the Arc Gauge. You can read more information in the [Dimensions]({%slug common-features/dimensions%}) article. |

You can also set the Gauge size in percentage values so it occupies its container when it renderes. If the parent container size changes, you must call the gauge's `Refresh()` C# [method](#methods) after the DOM has been redrawn and the new container dimensions are rendered.

## Arc Gauge Reference and Methods

> <style>
>     article style + table {
>         table-layout: auto;
>         word-break: normal;
>     }
> </style>

Reviewer: ```suggestion
@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
```
